src/libpgagroal/worker.c

```diff
@@ -159,6 +163,9 @@ pgagroal_worker(int client_fd, char* address, char** argv)
          /* The slot may have been updated */
          slot = client_io.slot;
       }
+
+      pgagroal_prometheus_client_active_sub();
+      pgagroal_prometheus_client_wait_add();
```

Comment:
I understand where you are coming from.

However, I don't think we need to put client shutdown into the `client_wait` category, since we are basically exiting shortly.

Hence we can remove `client_wait_add()` from here